pyproject.toml

```diff
@@ -91,7 +91,7 @@ venv = ".venv"
 venvPath = "."
 
 [tool.pytest.ini_options]
-addopts = "--benchmark-skip -m 'not hypothesis' -m 'not integration'"
+addopts = "-m 'not (integration or benchmark or hypothesis)'"
```

Comment:
1. codspeed doesn't work with --benchmark-skip because under the hood it does --benchmark-disable.
2. Multiple `-m`'s don't get combined, only the last one is captured.